server/functions.lua

```diff
@@ -63,7 +63,7 @@ end
 
 function QBCore.Functions.GetPlayers()
     local sources = {}
-    for k, v in pairs(QBCore.Players) do
+    for k, _ in pairs(QBCore.Players) do
```

Comment:
You can remove this underscore completely, it doesn't need to be there if it's not used.
In lua when you have a parameter that's not used after one that is used and there are none used after the one that's not used, then you can delete everything after the one that's used and has nothing that is used after it.

Reply:
It's a standard in Lua to use underscores as not-used variables.

Reply:
But instead of adding them, you can remove them if there is nothing else beong used after it, just like you can remove unused variables from the funtion arguments in events, no need for it

Reply:
I only use underscores when there is a variable after the underscore that's used, otherwise I remove it because it is something extra that serves no purpose basically

Reply:
```suggestion
    for k in pairs(QBCore.Players) do
```